agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -60,6 +64,28 @@ async def create_evaluation(
         if app is None:
             raise HTTPException(status_code=404, detail="App not found")
 
+        if await check_if_ai_critique_exists_in_list_of_evlautors_configs(
```

Comment:
I would wrap all of this under a function:
check_ai_critique_inputs to simplify the code